packages/react/src/hooks/participant-hooks.ts

```diff
@@ -35,9 +35,8 @@ export interface UseParticipantsOptions {
  * The useParticipants hook returns all participants (local and remote) of the current room.
  */
 export const useParticipants = (options: UseParticipantsOptions = {}) => {
```

Comment:
Removing this `useState` has no immediate effect because `useRemoteParticipants` catches the object-as-default-value problem. Still, I would argue that this is a future bug waiting to happen, and maybe the `useRemoteParticipants` hook will still get called instead of not getting called because the options object hasn't changed? 🤔

Reply:
In the latest commit that caching is present again, had the same thought!